pepagent/const.py

```diff
@@ -4,6 +4,10 @@
 ANNO_COL = "anno_info"
 NAMESPACE_COL = "namespace"
 NAME_COL = "name"
+TAG_COL = "tag"
 DIGEST_COL = "digest"
 
-DB_COLUMNS = [ID_COL, PROJ_COL, ANNO_COL, NAMESPACE_COL, NAME_COL, DIGEST_COL]
+DB_COLUMNS = [ID_COL, DIGEST_COL, PROJ_COL, ANNO_COL, NAMESPACE_COL, NAME_COL, TAG_COL]
+
+DEFAULT_NAMESPACE = "other"
```

Comment:
Should we maybe do `"_"`? Sort of how docker hub does it. Just a suggestion. I feel like `other` could be better.